packages/story-editor/src/components/panels/design/pageBackground/pageBackground.js

```diff
@@ -181,6 +204,7 @@ function PageBackgroundPanel({ selectedElements, pushUpdate }) {
           />
         </Row>
       )}
+      {failedContrast && <Warning message={COLOR_COMBINATION} />}
```

Comment:
Added the warning message here as well cause I figured why not 🤷🏻‍♀️

Reply:
I think it makes sense to put here. Maybe we should slightly alter the copy to make it specific to background color vs text color. May be a good question for @agingoldseco though:

```
Background: Some text on the page may be hard for people to read against this background. Update the background color to increase the contrast.

Text: This text may be hard for people to read. Update the color or the font size to increase the contrast.
```

Reply:
Happy to update copy to whatever y'all think is best. I am not great at copy so I have no preference.

Reply:
Hmph thinking about it now, I'm wondering if the background is a good candidate for this check or not. The primary scenario I'm thinking about is:
- you have 2 text elements, one has high enough contrast with the background, the other does not
- you select the background see that the has a warning that says it doesn't have enough contrast, and the user should change the color
- the user changes the color to better contrast with the original conflicting text, but now the background is conflicting with the other text element (that didn't throw an error before)
- to the user, they changed the color, but the warning just never went away

Reply:
^^ not sure if this is a likely scenario, but curious if it's one we should consider.

Reply:
anyway, I don't want to throw a wrench in this PR which has already gotten design approval. I think if we find ourselves running into the scenario above more frequently, we can always do a little followup PR. Just some behavior to keep our eyes out for in the future

Reply:
Yeah I had thought about that too. My thinking was that regardless of which text element is causing the error, there is still an error and they should find a background that is suitable for all text if they don't want to see the message. The copy here was updated to `This background color may make some text hard to read.` Which I thinks helps convey that _some_  not necessarily all text may be hard to read.